### PR TITLE
Sophia - createFiles

### PR DESCRIPTION
**Pull request by Sophiashe88**
--create new files name

### CODE REVIEW

**Review by may-jia-0528 — COMMENTED** [review]
LGTM!!!

**Comment by may-jia-0528** [comment]
L

File: SophiaS/index.html [hunk 1]
```diff
@@ -16,4 +16,5 @@ <h1>
     <h2>
         Hello everyone, this is Sophia, I like go to shopping and find popular styles each year.
     </h2>
+    </h3> I want to create a new branch.
```
Comment (Muradil-Erkin) [inline SophiaS/index.html:19]: @Sophiashe88 h3 element has to have an opening tag `<h3>` and a closing tag `</h3>`, please add it.

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by may-jia-0528 — COMMENTED** [review]
Thank you!

**Comment by may-jia-0528** [comment]
Have a good night!

File: SophiaS/index.html (hunk 1 above)
Comment (may-jia-0528) [reply SophiaS/index.html:19]: Thank you!

**Pull request closed without merging**